### PR TITLE
[analytics] Decorate outbound links

### PR DESCRIPTION
**Pull request by chutten**
Captures left-button clicks on anchors and their descendants that look like they'd result in a navigation to another host.

### CODE REVIEW

**Comment by chutten** [comment]
Fixes #355

File: analytics/analytics.js [hunk 1]
```diff
@@ -24,6 +24,33 @@ if (window.trackingEnabled()) {
...
+    });
+  }
+}
+
+function findAnchor(el) {
+  while (el && el.nodeName != 'A') {
+    el = el.parentElement;
+  }
+  return el;
 }
```
Comment (wlach) [inline analytics/analytics.js:54]: nit: this line looks misaligned by one character

**Review by wlach — APPROVED** [review]
lgtm

File: analytics/analytics.js (hunk 1 above)
Comment (chutten) [reply analytics/analytics.js:54]: It's fine in the source file. I blame github's UI. :)

**Pull request merged**